tests/conftest.py

```diff
@@ -3,10 +3,14 @@
 from pathlib import Path
 
 import pytest
+from ape.contracts import ContractContainer
 from ape.utils import ZERO_ADDRESS
+from ethpm_types import ContractType
 
 from ape_safe.accounts import SafeAccount
 
+projects_directory = Path(__file__).parent / "contracts"
```

Comment:
you may wanna call this `contracts_directory` instead since typically in Ape projects, the project path contains the contracts folder